src/briefcase/commands/dev.py

```diff
@@ -192,6 +201,29 @@ def get_environment(self, app: AppConfig):
 
         return env
 
+    def venv_path(self, appname: str) -> Path:
+        """Return the path for the app's virtual environment.
+
+        :param app: The app config
+        :returns: Path where the venv should be located
+        """
+        return self.base_path / ".briefcase" / appname / "venv"
```

Comment:
This is the right approach for refactoring this; three followup suggestions:
1. We can use a venv name that reflects usage (e.g., `dev`), rather than a generic `venv`
2. The web dev command should override this, with a `dev-web` environment name
3. It might be worth modifying this so it's *just* the environment name that is customised on a per-platform basis. After all, the parent directory of the venv will always be the same - it's only the specific environment name that will change. That would also make this a property, as there's nothing app-specific about the return value.